Multisig/Data/Services/Utils/GSError.swift

```diff
@@ -340,4 +340,12 @@ enum GSError {
         let loggable = true
     }
 
+    struct ThirdPartyError: DetailedLocalizedError {
+        let description = "Third party library error"
+        let reason: String
+        let howToFix = "Please try again later or contact Safe support if this issue persists"
```

Comment:
Just to make sure, are you also updating the spec doc to include the new error reasons & codes?

Reply:
Yes, I do